src/main/java/spring/aspect/HandlerAspect.java

```diff
@@ -1,10 +1,55 @@
 package spring.aspect;
 
-import org.aspectj.lang.annotation.Aspect;
-import org.aspectj.lang.annotation.Pointcut;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.aspectj.lang.JoinPoint;
+import org.aspectj.lang.ProceedingJoinPoint;
+import org.aspectj.lang.annotation.*;
 import org.springframework.stereotype.Component;
 
+
 @Component
 @Aspect
 public class HandlerAspect {
+    private final static Logger LOG = LogManager.getLogger(HandlerAspect.class);
+
+    @Pointcut("execution(* spring.hander.IHandler.process(..))")
```

Comment:
You can reuse pointcut for multiply advices, it's not needed to setup new pointcut